h2o-py/h2o/model/binomial_uplift.py

```diff
@@ -50,39 +50,39 @@ def auuc(self, train=False, valid=False, metric=None):
 
     def auuc_normalized(self, train=False, valid=False, metric=None):
```

Comment:
is it too late to change order of parameters?
Other metrics methods having a `metric` param tend to put it first as it's usually the most significant param. Also surprised not to see the `xval` param as for other metrics functions, doesn't uplift support xval? I don't see anything in `_delegate_to_metrics` suggesting that we couldn't support `xval` param, but maybe it would break somewhere else...
Therefore expecting sth like:
```python
def auuc(self, metric=None, train=False, valid=False, xval=False):
```

Reply:
how was it working before with both `auuc` (and some siblings) in `ModelBase` AND `H2OBinomialUpliftModel`?
I guess the version in `ModelBase` was just overridden, correct?

Reply:
> I guess the version in ModelBase was just overridden, correct?

Yes, `ModelBase` is overridden.

Reply:
> Also surprised not to see the xval param as for other metrics functions, doesn't uplift support xval?

Xval is for crossvalidation? Uplift is not support cross validation (folds) yet

Reply:
Order of parameters will be handed here: https://h2oai.atlassian.net/browse/PUBDEV-8702